addons/survey/controllers/survey_session_manage.py

```diff
@@ -112,6 +112,39 @@ def survey_session_next_question(self, survey_token, **kwargs):
         else:
             return False
 
+    @http.route('/survey/session/survey_get_next_section_id', type='json', auth="public", website=True, sitemap=False)
+    def survey_get_next_section_id(self, survey_token, current_question_id=0):
```

Comment:
Not sure to understand why we have a get next section in addition to next question ... isn't it possible to simplify / merge code / decide to have one single point for all this computation ?